zhaquirks/tuya/__init__.py

```diff
@@ -1313,6 +1313,20 @@ def update_attribute(self, attr_name: str, value: Any) -> None:
         return self._update_attribute(attr.id, value)
 
 
+class TuyaPowerConfigurationCluster2AAA(PowerConfiguration, TuyaLocalCluster):
+    """PowerConfiguration cluster for devices with 2 AAA."""
+
+    BATTERY_SIZES = 0x0031
+    BATTERY_QUANTITY = 0x0033
+    BATTERY_RATED_VOLTAGE = 0x0034
+
+    _CONSTANT_ATTRIBUTES = {
+        BATTERY_SIZES: 4,
+        BATTERY_QUANTITY: 2,
+        BATTERY_RATED_VOLTAGE: 15,
+    }
+
+
```

Comment:
Please, put the class definition next to `TuyaPowerConfigurationCluster2AA` and `TuyaPowerConfigurationCluster3AA` classes.

Reply:
Moved up with dependency `TuyaLocalCluster`. 
------

The power clusters inherited from different bases
`class TuyaPowerConfigurationCluster2AA(TuyaPowerConfigurationCluster)`
and
`class TuyaPowerConfigurationCluster2AAA(PowerConfiguration, TuyaLocalCluster)`

Added some minor refactoring to bring `TuyaPowerConfigurationCluster` across to inheriting from `TuyaLocalCluster` (it already used attribute name to get the id)